src/main/app/claims/claimStoreClient.ts

```diff
@@ -73,8 +73,8 @@ export default class ClaimStoreClient {
           Authorization: `Bearer ${userAuthToken}`
         }
       })
-      .then((claims: object[]) => {
-        return claims.map((claim: object) => new Claim().deserialize(claim))
+      .then((claim) => {
+        return new Claim().deserialize(claim)
```

Comment:
Won't this return a list if you use a solicitor reference?

Reply:
we have validation to make sure they only put in a valid claim reference number. Searching by their reference number is not happening yet